CMSgov/design-system
# [WNMGDS-2249] Remove Mgov deprecated colors

## Summary

- Mgov had some semantic colors marked for deprecation and they have been removed
- Documentation has been updated 

## How to test

1. Run the doc site with `yarn start` and visit the migration page to see the additional non-semantic colors for Mgov that are being removed 

## Checklist




### If this is a change to code:


### If this is a change to documentation:
